web/utils/server-status-context.tsx

```diff
@@ -112,17 +116,34 @@ const ServerStatusProvider: FC<ServerStatusProviderProps> = ({ children }) => {
   const getStatus = async () => {
     try {
       const result = await fetchData(STATUS);
-      setStatus({ ...result });
+
+      if (result instanceof Error) {
+        throw result;
+      }
+
+      setStatus({ ...result, error: { type: null, msg: null } });
     } catch (error) {
+      setStatus(initialStatus => ({
+        ...initialStatus,
+        error: {
+          type: 'OWNCAST_SERVICE_UNREACHABLE',
+          msg: 'Cannot connect to our servers. Some functionality may not work correctly',
```

Comment:
Adding some specificity around "our servers" may make sense, especially to somebody not completely familiar with what's going on.  Plus the plural of "server" implies there's multiple servers.

[This](https://github.com/owncast/owncast/blob/develop/web/components/stores/ClientConfigStore.tsx#L39) is the error for the frontend. While not perfect, perhaps it might be a guide for what the admin error might look like.

Reply:
Sure. Updating the message to 
`Cannot connect to the Owncast service. Please check your internet connection or if needed, double check this Owncast server is running.`.
Does this work?

Reply:
On second thought, the message when viewed on the FatalModal seems a little odd in my opinion.
![image](https://user-images.githubusercontent.com/20909078/224492102-9071efe7-167e-457c-b8f7-d4382d05eb07.png)

Another alternative is?
`Cannot connect to the Owncast service. Please check you are connected to the internet and the Owncast server is running.`
![image](https://user-images.githubusercontent.com/20909078/224492215-455bbf1a-4070-4082-bd38-7cc1b3b8773a.png)

Which do you like more?

Reply:
That works for me!